go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.13.0
 	sigs.k8s.io/yaml v1.3.0
 	stathat.com/c/consistent v1.0.0
-	volcano.sh/apis v1.10.0-alpha.0.0.20240709084748-78d912ce096c
+	volcano.sh/apis v0.0.0-00010101000000-000000000000
```

Comment:
This apis version is so weird

Reply:
Already change to https://github.com/volcano-sh/apis/pull/135